bundle/edu.gemini.p1monitor/src/main/scala/edu/gemini/p1monitor/config/P1MonitorConfig.scala

```diff
@@ -103,7 +103,6 @@ class P1MonitorConfig(ctx: BundleContext) {
   private def toTemplate(s: String): P1PDF.Template = s match {
     case "ar" | "br" | "kr" | "uh" => P1PDF.GeminiDefault
     case "cfh" | "subaru" | "keck" => P1PDF.GeminiDefault
-    case "dt" | "pw"               => P1PDF.GeminiDefault
```

Comment:
I'm confused about this change. I don't think it is related to NGS2 and it doesn't appear in either the `develop` or `release` branch so I will put this back unless I hear otherwise.

Reply:
This seems unrleated, P1Monitor has no business with NGS

Reply:
Reset to original.